test/micro/org/openjdk/bench/jdk/internal/jrtfs/ImageReaderBenchmark.java

```diff
@@ -220,858 +312,891 @@ private static void reportMissingClassesAndFail(ColdStart state, int errors) thr
-            "/modules/java.base/java/util/stream/FindOps$FindOp.class",
-            "/modules/java.base/java/util/Spliterators.class",
-            "/modules/java.base/java/util/Spliterators$IteratorSpliterator.class",
-            "/modules/java.base/java/lang/module/Configuration.class",
-            "/modules/java.base/java/lang/module/Resolver.class",
-            "/modules/java.base/java/lang/ModuleLayer.class",
-            "/modules/java.base/java/util/SequencedSet.class",
-            "/modules/java.base/java/util/LinkedHashSet.class",
-            "/modules/java.base/java/util/SequencedMap.class",
-            "/modules/java.base/java/util/LinkedHashMap.class",
-            "/modules/java.base/java/lang/module/ResolvedModule.class",
-            "/modules/java.base/jdk/internal/module/ModuleLoaderMap$Mapper.class",
-            "/modules/java.base/jdk/internal/loader/AbstractClassLoaderValue$Memoizer.class",
-            "/modules/java.base/jdk/internal/module/ServicesCatalog$ServiceProvider.class",
-            "/modules/java.base/java/util/concurrent/CopyOnWriteArrayList.class",
-            "/modules/java.base/java/lang/ModuleLayer$Controller.class",
-            "/modules/java.base/jdk/internal/module/ModuleBootstrap$SafeModuleFinder.class",
-            "/modules/java.base/jdk/internal/vm/ContinuationSupport.class",
-            "/modules/java.base/jdk/internal/vm/Continuation$Pinned.class",
-            "/modules/java.base/sun/launcher/LauncherHelper.class",
-            "/modules/java.base/sun/net/util/URLUtil.class",
-            "/modules/java.base/jdk/internal/loader/URLClassPath$Loader.class",
-            "/modules/java.base/jdk/internal/loader/URLClassPath$FileLoader.class",
-            "/modules/java.base/jdk/internal/loader/Resource.class",
-            "/modules/java.base/java/io/FileCleanable.class",
-            "/modules/java.base/sun/nio/ByteBuffered.class",
-            "/modules/java.base/java/security/SecureClassLoader$CodeSourceKey.class",
-            "/modules/java.base/java/security/PermissionCollection.class",
-            "/modules/java.base/java/security/Permissions.class",
-            "/modules/java.base/java/lang/NamedPackage.class",
-            "/modules/java.base/jdk/internal/misc/MethodFinder.class",
-            "/modules/java.base/java/lang/Readable.class",
-            "/modules/java.base/java/nio/CharBuffer.class",
-            "/modules/java.base/java/nio/HeapCharBuffer.class",
-            "/modules/java.base/java/nio/charset/CoderResult.class",
-            "/modules/java.base/java/util/IdentityHashMap$IdentityHashMapIterator.class",
-            "/modules/java.base/java/util/IdentityHashMap$KeyIterator.class",
-            "/modules/java.base/java/lang/Shutdown.class",
-            "/modules/java.base/java/lang/Shutdown$Lock.class");
+    /// Note: This list is inherently a little fragile and may end up being more
```

Comment:
Yes, this kind of test/benchmark will bit rot very fast.
The existing startup measurements are more stable and reflect actual startup times, not a simulation thereof.